### PR TITLE
Use https for all links

### PR DESCRIPTION
**Pull request by dns2utf8**
Hi all

This is a followup to https://github.com/gtk-rs/gtk/pull/1054

Cheers,
Stefan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 Pango bindings for Rust.
 
-- [Gtk-rs project site](http://gtk-rs.org/)
+- [Gtk-rs project site](https://gtk-rs.org/)
 
-- [Online documentation](http://gtk-rs.org/docs/)
+- [Online documentation](https://gtk-rs.org/docs/)
```
Comment (GuillaumeGomez) [inline README.md:7]: Invalid link.

**Comment by dns2utf8** [comment]
Fixed

**Comment by dns2utf8** [comment]
I am not sure what happend to `src/attr_list.rs` since I changed one character not the complete file...

**Comment by GuillaumeGomez** [comment]
`src/attr_list.rs` was removed so please don't bring it back from the dead haha.

**Comment by dns2utf8** [comment]
Ahh, okay :smile: removing it fixed the weird diff

**Comment by GuillaumeGomez** [comment]
Sorry, I was completely wrong, the file is still there apparently... I wonder what happened in the previous diff then... So, please only change the URL in the header of the file I guess XD (and sorry for the misunderstanding...). Also, if you didn't do it, I suggest you to rebase on `master`, maybe the weird things come from the `master` branch?

**Comment by dns2utf8** [comment]
I pulled this master and copied the file manually. Now it works as expected and the GH diff looks okay too.
Not sure why it went so wrong the last time :shrug: :smile:

**Comment by GuillaumeGomez** [comment]
Dark magic. 😆 

Thanks!

**Pull request merged**